### ISSUE TITLE
john paller wants more kudos awards for ethdenver

### ISSUE THREAD

**Issue by owocki** [gitcoinco/kudosbadges#142]
List of awards he wants:

ETHDenver 2019: Spirit of #BUIDL
ETHDenver 2019: Bufficorn's Best
ETHDenver 2019: People's Choice
ETHDenver 2019: Best Media Headline
ETHDenver 2019: Best for Ethereum
ETHDenver 2019: Highest "WOW" Factor
ETHDenver 2019: Most Innovative
ETHDenver 2019: Adoption Driver
ETHDenver 2019: Runner Up
ETHDenver 2019: Impact Track Finalist
ETHDenver 2019: Open Track Finalist

These are going to be one color (not the full color ones).  They will come out as trophies that look like this:

![kudos-samplev1](https://user-images.githubusercontent.com/513929/51686748-31b18980-1fae-11e9-8f88-0075ae0d878b.jpg)

We may be able to save some time by making each artwork similar to each other..  maybe each of them has a bufficorn up on stage in a tux clutching some award, and that award (and maybe the facial expression / orientation of the bufficorn) is different per kudos?  

We could use the shadow bufficorn and the bufficorn head for different things.

**Comment by MladenPetronijevic** [comment]
![image](https://user-images.githubusercontent.com/41165756/51773648-c7a0fd80-20ef-11e9-8977-a0a3cf97b76d.png)

**Comment by owocki** [comment]
ok i talked to john

for all 10 of the images, we were thinking that we’d do 10 small permutations on top of the above image..

- some where bufficorn is on the right, some on left.
- switch out the trophy (and image on the trophy) according to which trophy its for.
- maybe change the bufficorns facial expression / clothes
- You could also use the other bufficorn designs that are in the folder. Jumping, robot, dancing, etc.

and so that way the trophies are similar but also slightly different..

does that work?

**Comment by sriharikapu** [comment]
Is there a bounty on this.? Can I help in the design? @owocki 
![statuso](https://user-images.githubusercontent.com/6247133/51805573-fe227800-22a9-11e9-9edf-fc6d95f12633.png)

![statusi](https://user-images.githubusercontent.com/6247133/51805576-0bd7fd80-22aa-11e9-8834-f843a24cd37b.png)

**Comment by owocki** [comment]
@sriharikapu theres no bounty no, 

1. have you seen the design style guide at https://github.com/gitcoinco/kudosbadges
2. i like what you've done so far :) but we've already got a designer slated for this 
3. does any of the tickets in the todo column of https://github.com/gitcoinco/kudosbadges/projects/1 look good to you?

**Comment by MladenPetronijevic** [comment]
![image](https://user-images.githubusercontent.com/41165756/51853134-da9e1280-2327-11e9-9750-303ac561ebd2.png)

**Comment by owocki** [comment]
feedback from john

1. change 'ethdenver' to say ETHDENVER '19
2. lets add a little variety to what the bufficorn is doing.  you can use the assets in the drive folder ( Nerd guy, tophat/monicle, music, jumping, etc. )

**Comment by MladenPetronijevic** [comment]
![untitled](https://user-images.githubusercontent.com/41165756/51870185-24e8b900-2353-11e9-8b93-2ab1150a6486.png)

**Comment by senvik** [comment]
Love the variety! The Open Track Finalist and Impact Track Finalist are almost like sister awards for the top finishers in each category. Does it make sense for them to have the same Bufficorn design?

**Comment by MladenPetronijevic** [comment]
like this?
![untitled](https://user-images.githubusercontent.com/41165756/52050254-44990080-2550-11e9-8146-69f1aef2570a.png)

**Comment by senvik** [comment]
Yeah! Looks awesome. Those two should be the only ones that have that design. Thank you!

**Comment by owocki** [comment]
cool @senvik @MladenPetronijevic can we get the final svgs this week?  any other changes needed from here?
